examples/integrations/coherence/helidon-creditscore-mp/pom.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Shouldn't this go under `examples/microprofile` ?

The only exception we have currently is the quickstarts.

How about keeping it simple for now, and just have this be the coherence example. We can rethink the layout when/if we add more.

I.e. `examples/microprofile/coherence`

Reply:
I plan on adding an SE version of this on the heels of this PR. I'd prefer to leave this where it is.